### PR TITLE
Nicolas' quit button

### PR DESCRIPTION
**Pull request by jnburchett**
As titled.  Also added patch in convolvecos for empty cfg.fitidx

### CODE REVIEW

**Comment by ntejos** [comment]
Just tested it on my computer, and it breaks once pressed. It seems is not quitting properly...

File: joebvp/makevoigt.py [hunk 1]
```diff
@@ -122,7 +122,7 @@ def convolvecos(wave,profile,lines,zs):
             fitwaves=wave[cfg.fitidx]
         else:
             # todo: figure out why are there cfg.fitidx = []?
-            pass
+            fitwaves = wave
             # import pdb; pdb.set_trace()
```
Comment (ntejos) [inline joebvp/makevoigt.py:126]: Are you sure this is what we want? i.e. for the case of empty `cfg.fitidx=[]`, `fitwaves` will become the full wavelength array...

**Review by ntejos — CHANGES_REQUESTED** [review]

File: joebvp/makevoigt.py (hunk 1 above)
Comment (jnburchett) [reply joebvp/makevoigt.py:126]: Can you recommend something better?

Comment (ntejos) [reply joebvp/makevoigt.py:126]: Nop, but not sure if this will be used at all. If you leave it with "pass" it may just work the same.

**Review by ntejos — APPROVED** [review]

**Comment by ntejos** [comment]
I think I solved the problem now. Please review my latest commit.

**Comment by ntejos** [comment]
@jnburchett are we good with this?

**Comment by jnburchett** [comment]
The GUI now takes longer to load, which I suspect is due to one of my changes, but we have addressed the issues at hand.  Merging.

**Pull request merged**